web/src/components/meetings/MeetingListHeader.vue

```diff
@@ -38,21 +47,42 @@ export default {
     pastMeetingCount: Number
   },
   data: () => ({
-    // TODO: change the index to 0 after changing list order to NEWEST_FIRST
     selectedSortOptionIndex: 1,

```

Comment:
Change this to **2** to allow the 'NEWEST_FIRST' to be the first option

Reply:
fixed